shell/browser/native_window_views.cc

```diff
@@ -1318,6 +1342,51 @@ bool NativeWindowViews::IsVisibleOnAllWorkspaces() {
   return false;
 }
 
+void NativeWindowViews::SetVibrancy(const std::string& type) {
```

Comment:
```suggestion
void NativeWindowViews::SetVibrancy(base::StringPiece type) {
```

The API code we call below allows a StringPiece, so we should use that more flexible type here too